distributed/tests/test_worker.py

```diff
@@ -1677,41 +1677,40 @@ async def test_story_with_deps(c, s, a, b):
     Assert that the structure of the story does not change unintentionally and
     expected subfields are actually filled
     """
-    dep = c.submit(inc, 1, workers=[a.address])
-    res = c.submit(inc, dep, workers=[b.address])
+    dep = c.submit(inc, 1, workers=[a.address], key="dep")
+    res = c.submit(inc, dep, workers=[b.address], key="res")
```

Comment:
I tweaked two tests in this module to use hardcoded keys, as they make the output of assert_worker_story a lot more readable.